src/Hedgehog/Plutus/Model.hs

```diff
@@ -0,0 +1,62 @@
+{-# OPTIONS_GHC -Wno-missing-import-lists #-}
+
+-- I think they are fine here because they are just reexported
```

Comment:
Unfortunately style guide says no. You may use wildcard exports, though (these are not allowed for imports)

Reply:
Fair enough :+1: